src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  grid-auto-rows: 300px;
+  justify-content: center;
+  gap: 48px;
+  max-width: 944px;
+  margin: 0 auto;
```

Comment:
You can remove `margin` using `justify-content`

Reply:
Thank you for all your advice!)